src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -2,6 +2,10 @@
 //! with the goal of keeping developers synchronized with important modifications in
 //! the bootstrap.
 
+#[cfg(test)]
+#[path = "../tests/change_tracker.rs"]
```

Comment:
Why are we doing this non-standard test movement? Just put the file in src/bootstrap/src/utils/change_tracker/test.rs -- that is the canonical way to add unit tests.

Reply:
I followed the current approach in other bootstrap tests, but I agree. Will update it along with others.